lib/slots.ts

```diff
@@ -1,137 +1,58 @@
 import dayjs, { Dayjs } from "dayjs";
-import timezone from "dayjs/plugin/timezone";
+import isBetween from "dayjs/plugin/isBetween";
 import utc from "dayjs/plugin/utc";
 
-dayjs.extend(utc);
-dayjs.extend(timezone);
+import { WorkingHours } from "./types/schedule";
 
-type WorkingHour = {
-  days: number[];
-  startTime: number;
-  endTime: number;
-};
+dayjs.extend(utc);
+dayjs.extend(isBetween);
 
-type GetSlots = {
+export type GetSlots = {
   inviteeDate: Dayjs;
   frequency: number;
-  workingHours: WorkingHour[];
-  minimumBookingNotice?: number;
-  organizerTimeZone: string;
+  workingHours: WorkingHours[];
+  minimumBookingNotice: number;
 };
 
-type Boundary = {
-  lowerBound: number;
-  upperBound: number;
+const getMinuteOffset = (date: Dayjs, step: number) => {
+  // Diffs the current time with the given date and iff same day; (handled by 1440) - return difference; otherwise 0
+  const minuteOffset = Math.min(date.diff(dayjs().startOf("day"), "minutes"), 1440) % 1440;
```

Comment:
modulus is used here but not in other places - modulus seems like a really good use for this sort of thing!

Reply:
Suggestions on other places to use modulus definitely appreciated, it was a nobrainer (well, not really, I did have to think about how to do this without writing if statements) - but it was a nobrainer to include the % in this situation.